modules/media/help/media.md

```diff
@@ -0,0 +1,17 @@
+# Media
+
+The Media module serves as a repository for files associated with a particular candidate-timepoint or instrument in a study.  Users can access and upload files such as PDF scans, recordings, log files, or stimulus presentation media in formats such as .pdf, .mp4, .mp3, and .txt. 
+
+## Searching for a Media File
+Under the Browse tab, use the Selection Filters to search for files by fields such as including filename, file type, candidate PSCID, Visit Label, Instrument, Site, or name of user who uploaded the file.  Partial string matching can be used on many fields (e.g. filename).  As filters are selected, the below data table will dynamically update with relevant results.  Click the “Clear Filters” button to reset all filters. 
+
+Within the data table, results can be sorted in ascending or descending order by clicking on any column header. To download a file, click on the filename (in blue text). 
+
+## Uploading a Media File
+Under the “Upload” tab, users will be able to upload a new media file. 
+Note that a file to be uploaded should follow the naming convention listed at the top of the Upload page: **[PSCID]****_****[VisitLabel]****_****[Instrument]**
+
+Users must specify information about the file by selecting from a number of dropdown menus including PSCID, Visit Label, Site, Instrument (optional), Date of Administration, and enter comments on the file in the textbox provided. Lastly, users will select the file they wish to upload by clicking the “Browse” button and selecting the file from their computer. To begin the upload, users must click “Upload File”. 

```

Comment:
I don't think "a number of" is required in this sentence. "by selecting from dropdown menus" flows better.. (also same issues with capitalization).

"Lastly, users will select" is also kind of awkward. Maybe just "Next, users select.." or something of the sort would flow better.